editor/plugins/org.fusesource.ide.wsdl2rest.ui/readme.md

```diff
@@ -0,0 +1,39 @@
+# Wsdl2Rest Wizard Code
+Last update: 8-DEC-2017
+
+This plug-in provides a wizard wrapper for the Wsdl2Rest code originally found here: [https://github.com/jboss-fuse/wsdl2rest](https://github.com/jboss-fuse/wsdl2rest)
+
+## Running the wizard
+NOTE: THIS DOES NOT WORK CURRENTLY, BUT DEMONSTRATES THE BASIC APPROACH.
+
+In the project you will find an "example" directory that includes several files - a couple of classes and a wsdl file.
+
+Create a new Fuse Integration project. Leave it as a Blank project that uses the Spring DSL.
+Create a new src/main/resources/wsdl directory.
+Copy the echoService.wsdl file into the src/main/resources/wsdl directory.
+
+Select the project, click New->Other, and select the JBoss Fuse->WSDL to Camel Rest DSL Wizard. 
+
+On the "Select Incoming WSDL and Project for Generated Output" page, click the "..." button beside the WSDL File field and select echoService.wsdl.
+Click Next.
+On the "Specify Advanced Options for wsdl2rest Processing", click the "..." beside the Bean Class field and select the EchoServiceImpl class. 
+Click Finish.
+
+Select the project and refresh it to see the generated code and new rest-camel-context.xml file. 
```

Comment:
I say "...to see the generated code and new rest-camel-context.xml file." The second "the" you included is implied because it's a simple list of two items. You could certainly include it, but it's not necessary.